### PR TITLE
fix index.html :popup snapshoot-1.png

### PR DESCRIPTION
**Pull request by andym80**
This case is for check pop-up functionality. Then will be converted into the required image and style .

### CODE REVIEW

File: navbar.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//   nav.classList.add('active');
+// });
+
+// close.addEventListener('click', () => {
+//   nav.classList.remove('active');
+// });
+
+// navLink.forEach((element) => element.addEventListener('click', () => {
+//   nav.classList.remove('active');
+// }));
```
Comment (OmarRamoun) [inline navbar.js]: - [ ] [REQRUIED] Kindly, try not to use commented code in your code base. It makes the whole code base unclean and hard to maintain. Instead you can use `git reset` for that manner.

File: navbar.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+function classToggle() {
+  const navs = document.querySelectorAll('.Navbar__Items')
+
+  navs.forEach(nav => nav.classList.toggle('Navbar__ToggleShow'));
```
Comment (OmarRamoun) [inline navbar.js]: - [ ] [REQUIRED] Kindly, make sure to match the class names with your html. In order for your code to work properly.

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @andym80 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @andym80 , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**